docs/0install/setup.md

```diff
@@ -0,0 +1,28 @@
+The 0install distribution of the client is to allow zanata-cli to be available on multiple platform. This distribution requires [0install](http://0install.net/) to run.
+
+## Prerequisites
+
+1. Install [0install](http://0install.net/injector.html) on your machine.
```

Comment:
The advantage of having single feeds file is when the dependency is fix, mac user just need to do  0install update zanata-cli. If separated,  we will need to force them to change url again.